src/typescript/sdk/src/indexer-v2/types/json-types.ts

```diff
@@ -397,6 +412,7 @@ export type DatabaseStructType = {
   ChatEventData: ChatEventData;
   StateEventData: StateEventData;
   GlobalStateEventData: GlobalStateEventData;
+  Candlestick: CandlestickData;
```

Comment:
```suggestion
  NormalCandlestick: NormalCandlestickData;
```

just to keep things consistent

Reply:
Resolving bc we are using `candlesticks` as the table name now